python/cuml/prims/stats/covariance.py

```diff
@@ -102,6 +133,9 @@ def cov(x, y, mean_x=None, mean_y=None, return_gram=False, return_mean=False):
             "X and Y must have same shape %s != %s" % (x.shape, y.shape)
         )
 
+    if x is y and cupyx.scipy.sparse.issparse(x):
+        return _cov_sparse(x, return_gram=return_gram, return_mean=return_mean)
```

Comment:
Please add a comment about the motivation for this delegation here. Ideally with a link to the corresponding issue.